configure.ac

```diff
@@ -339,36 +339,29 @@ EXTRA_WARNINGS=""
 
 WARNLIST="
 	all
+	extra
+	unused
 	shadow
 	missing-prototypes
 	missing-declarations
+	suggest-attribute=noreturn
+	suggest-attribute=format
 	strict-prototypes
-	declaration-after-statement
```

Comment:
This is only really relevant when attempting to keep C89 compatibility. C99 has been enabled, and so this is no longer needed.